CMakeLists.txt

```diff
@@ -0,0 +1,53 @@
+cmake_minimum_required(VERSION 3.4)
+project(linenoise C)
+
+include(GNUInstallDirs)
+
+option(LINENOISE_BUILD_EXAMPLE OFF)
+
+# -----------------
+# LIBRARY
+# -----------------
+add_library(${PROJECT_NAME} linenoise.c)
+target_include_directories(${PROJECT_NAME}
+    INTERFACE $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/linenoise>
```

Comment:
Use `install(TARGETS ... INCLUDES DESTINATION)` instead for specifying includes for the install.